src/chocolatey/infrastructure.app/nuget/NugetList.cs

```diff
@@ -54,6 +54,33 @@ public static IEnumerable<IPackage> GetPackages(ChocolateyConfiguration configur
                         .Where(p => configuration.Prerelease || p.IsReleaseVersion())
                         .distinct_last(PackageEqualityComparer.Id, PackageComparer.Version);
         }
+
+        public static int GetCount(ChocolateyConfiguration configuration, ILogger nugetLogger)
+        {
+            var packageRepository = NugetCommon.GetRemoteRepository(configuration, nugetLogger);
+            IQueryable<IPackage> results = packageRepository.Search(configuration.Input, configuration.Prerelease);
+
+            if (configuration.AllVersions)
+            {
+                return results.Where(PackageExtensions.IsListed).Count();
+            }
+
+            if (configuration.Prerelease && packageRepository.SupportsPrereleasePackages)
+            {
+                results = results.Where(p => p.IsAbsoluteLatestVersion);
+            }
+            else
+            {
+                results = results.Where(p => p.IsLatestVersion);
+            }
+
+            return results
+                        .AsEnumerable()
+                        .Where(PackageExtensions.IsListed)
+                        .Where(p => configuration.Prerelease || p.IsReleaseVersion())
+                        .distinct_last(PackageEqualityComparer.Id, PackageComparer.Version)
+                        .Count();
+        }

```

Comment:
If they specified paging, then Count() would be inaccurate. The though process here is you could use the same config for both `List` and `Count`, and count would return the correct result.